test/config/featureSwitches/FeatureSwitchingSpec.scala

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config.featureSwitches
+
+import base.SpecBase
+import config.AppConfig
+import org.mockito.Matchers.any
+import org.mockito.Mockito.{mock, reset, when}
+import org.scalatest.BeforeAndAfterAll
+import play.api.Configuration
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+
+import java.time.LocalDate
+
+class FeatureSwitchingSpec extends SpecBase with BeforeAndAfterAll {
+  val mockConfig = mock(classOf[Configuration])
+  val mockServicesConfig = mock(classOf[ServicesConfig])
+  val config: AppConfig = new AppConfig(mockConfig, mockServicesConfig)
+
+  class Setup {
+    reset(mockConfig)
+    reset(mockServicesConfig)
+    val featureSwitching: FeatureSwitching = new FeatureSwitching {
+      override implicit val appConfig: AppConfig = config
+    }
+    sys.props -= CallAPI1812ETMP.name
+    sys.props -= UseAPI1812Model.name
+    sys.props -= featureSwitching.TIME_MACHINE_NOW
+  }
+
+  override protected def afterAll(): Unit = {
+    super.afterAll()
+    sys.props -= CallAPI1812ETMP.name
+    sys.props -= UseAPI1812Model.name
```

Comment:
Could you add 
```suggestion
    sys.props -= featureSwitching.TIME_MACHINE_NOW
```
It will remove the time machine date ready for another spec

Reply:
Replace ` sys.props -= UseAPI1812Model.name` with the line you suggested or just add it below?

Reply:
Ah! Sorry! Yes, add it below please